odoo/tools/translate.py

```diff
@@ -79,6 +79,9 @@
 
 TRANSLATED_ATTRS.update({f't-attf-{attr}' for attr in TRANSLATED_ATTRS})
 
+TRANSLATE_FUNCS = {'default': True}
```

Comment:
This is strange, because you would think that it maps str to a function, but here it is a boolean.